### ISSUE TITLE
Cannot play songs from Library -> Songs

### ISSUE THREAD

**Issue by Kiritsu** [ciderapp/Cider-2#859]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [X] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [X] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### How did you download Cider?

Microsoft Store

### Operating System

Windows 

### Commit Hash

7b04e2c902aa2dab4065c276beb3302b22385240

### Build Timestamp

29/10/2024, 04:16:42

### Describe the Bug

When I try to play a songs from my library, in the "Songs" tab, nothing happens.

![image](https://github.com/user-attachments/assets/353a167b-e9a1-4bf5-a4ae-5cbb087babef)

After opening the DevTools, whenever I try to play a song, it says: 

![image](https://github.com/user-attachments/assets/655b0a1b-688c-46e1-8a15-1dfb4f7f50d2)

I can play songs from the other tabs successfully.

### Steps to Reproduce

- Open the Library
- Open to Songs tab
- Attempt to play any song from there

**Comment by Baustelle17** [comment]
I've had the same issue for a while. Can play songs from anywhere on the interface except for the library tab.

**Comment by CoderNoel** [comment]
I too am in the same boat for Cider on Mac. Would love to have it fixed as soon as possible.

**Comment by wjarka** [comment]
https://github.com/ciderapp/Cider-2/issues/856

Can you check if it's the same case as mine?

**Comment by Baustelle17** [comment]
It doesn't seem to be. For me, the entire library doesn't play. Not even the top shuffle button works.

**Comment by Baustelle17** [comment]
Not sure why, but the bug is fixed for me. It might have been changing storefront settings: Settings -> General -> Apple Music (at the bottom) -> Apple Music Storefront (dropdown, select your country). However I remember this issue randomly appearing and disappearing so I'm doubtful. See if it works.

**Comment by defaultunknown** [comment]
bump

**Comment by Roxsy-0** [comment]
any luck on resolving this issue? my friend has been going through the same problem and I've been trying to find fix for it.

**Comment by XCraftMC23** [comment]
After doing some investigation it seems that the songs listed in the library use a different webrequest URL ([amp-api.music.apple.com](amp-api.music.apple.com)) than the ones used on the front page ([play.itunes.apple.com](play.itunes.apple.com))

**Comment by JellyPeixe** [comment]
I'm facing the same issue in windows using the latest cider version (2.6.0.5).

**Comment by Baustelle17** [comment]
It's possibly an issue with queue length, when switching 'Songs' view to favorites only (pressing the star) the library becomes shuffle-able and playable for me. I have a similar issue where a playlist compiled with all the songs in my library doesn't list (it only doesn't show up on cider).

**Comment by Baustelle17** [comment]
bump

**Comment by coredev-uk** [comment]
Please confirm that you are on 2.6.0 and have your language settings set to auto in Settings > General > Apple Music. Make sure both the storefront and language are auto.

**Comment by Baustelle17** [comment]
Yes, I can confirm that the songs tab still does not work on version 2.6.0.15 with the storefront and language settings set to auto.

**Comment by Kiritsu** [comment]
Hello, 

Using version 2.6.0.15, it now works properly as intended, I haven't changed any settings. I'm not closing the issue since it seems to still affect some people.

Thanks

**Comment by theironraiden** [comment]
Hi,

This seems to be intermitently working for me. Switching to Widevine as DRM and disabling album art in songs library view seems to have improved it, but it might be placebo.

**Comment by nickmmendes** [comment]
bump
